### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by Dec-F** [zeit/hyper#3054]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: window 10 1803
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**:https://gist.github.com/Dec-F/af0f2d54fd25dbf4676dbf5ea0792416
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**:

**Comment by jcrben** [comment]
@Dec-F you didn't link to your hyper.js - instead you linked to the blog log which is good, but your config is also needed.

I also suggest trying this in a clean Windows VM if you need to use Windows.

**Comment by Dec-F** [comment]
@jcrben   Yes, I tried to run on the virtual machine's windows and it worked fine.Thanks

**Comment by jcrben** [comment]
@Dec-F did you figure out the ultimate cause of the problem?
